CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# dbt_iterable v0.RELEASE.RELEASE
+# dbt_iterable v0.11.1
+[PR #44](https://github.com/fivetran/dbt_iterable/pull/44) includes the following updates:
+
+## Bug Fixes
+- Introduces variable `iterable__using_event_extension` to disable the `event_extension` table and exclude its fields from persisting downstream. This allows the downstream models to run even if the source `event_extension` table does not exist. For more information on how to configure the `iterable__using_event_extension` variable, refer to the [README](https://github.com/fivetran/dbt_iterable/blob/main/README.md#step-4-enablingdisabling-models).
+- Persists `user_history` passthrough columns, as stipulated via the `iterable_user_history_pass_through_columns` variable, through to the `iterable__users` model. For more information on how to configure the `iterable_user_history_pass_through_columns` variable, refer to the [README](https://github.com/fivetran/dbt_iterable/blob/main/README.md#passing-through-additional-fields).
 
 ## Under the Hood
```

Comment:
You can add more details around the consistency tests you've created here too.

Reply:
added!